cmd/kubelet/app/options/options.go

```diff
@@ -41,11 +41,7 @@ const (
 	DefaultKubeletContainersDirName = "containers"
 )
 
-// KubeletServer encapsulates all of the parameters necessary for starting up
-// a kubelet. These can either be set via command line or directly.
-type KubeletServer struct {
-	componentconfig.KubeletConfiguration
-
+type KubeletFlags struct {
```

Comment:
nit: add a comment describing what should be in KubeletFlags, as opposed to `KubeletConfiguration`